--- ./keys/Makefile.org3	2006-02-01 16:04:28.999279992 +0100
+++ ./keys/Makefile	2006-02-01 16:05:21.561289352 +0100
@@ -14,9 +14,9 @@
 
 	$(COMPILE) gen-keys.c
 	$(BUILD) gen-keys gen-keys.o
-	strip -v gen-keys
+#	strip -v gen-keys
 #	./gen-keys 32 16 > keys.txt
-	chown root:root ./rpl.pl ./gen-keys
+#	chown root:root ./rpl.pl ./gen-keys
 	chmod 0700 ./rpl.pl ./gen-keys
 	./rpl.pl $(SYSTEM_ROOT)/gui/include/vhcs2-db-keys.php $(SYSTEM_ROOT)/engine/vhcs2-db-keys.pl
         
